containers/dockerfile_gpuhoomd2conda

```diff
@@ -0,0 +1,80 @@
+FROM nvidia/cuda:10.2-devel 
```

Comment:
is this file the same as `dockerfile_gpuhoomdconda`?

Reply:
Yeah, I just renamed the file to include the hoomd version. I thought usually github would recognize that and just mention that the change was renaming the file, but the changes make it look as if I also made `dockerfile_gpuhoomd2conda`

Reply:
It looks like I copied the old file and renamed it, so the old one is still in there. It should be removed now.

Reply:
if you use `git mv` it'll rename the file, but renaming that file (and the container it builds) could cause _a lot_ of downstream conflicts. (It's used as the base container for many repos.) So I would recommend not changing it.

Reply:
@jennyfothergill 

Maybe we should create something that keeps track of where these containers are used, even if it's just a text file that exists in this repo? I think potential downstream conflicts aren't necessarily a reason to avoid making changes that make sense, like identifying the hoomd version in a container file name, but should be fixed themselves when changes do occur (e.g. having 2 different hoomd version containers).